docs/plugins.jetbrains.com/general_usage_instructions.html

```diff
@@ -6,7 +6,7 @@
     Open Git Machete tab:
     <ul>
       <li>Navigate <b>Git</b> (tool window) -> <b>Git Machete</b> (tab)</li>
-      <li>Or use key shortcut: Ctrl + Alt + Shift + M</li>
+      <li>Or use key shortcut: Ctrl + Alt + Shift + M (Ctrl + Option + Shift + M on Mac OS)</li>
```

Comment:
Pls update on Marketplace site... do you have access there?

Reply:
Updated already
![image](https://user-images.githubusercontent.com/3383210/167031942-ea18826c-391a-497c-b42e-302885713c8d.png)

Reply:
![image](https://user-images.githubusercontent.com/19799111/167078145-c6c6bbe8-1c1f-4ea8-96fd-d32e154f8181.png)